x-pack/plugins/ml/public/application/routing/routes/datavisualizer/index_based.tsx

```diff
@@ -27,6 +27,9 @@ export const indexBasedRouteFactory = (
 ): MlRoute => ({
   id: 'data_view_datavisualizer',
   path: '/jobs/new_job/datavisualizer',
+  title: i18n.translate('xpack.ml.dataVisualizer.dataView.docTitle', {
+    defaultMessage: 'Data View Data Visualizer',
```

Comment:
I'm not sure about this one, with repetition of `Data`. How does `Data Visualizer | Data View` look, or even just `Data Visualizer` ?

Reply:
Or `Index data visualizer`?

Reply:
Index data visualizer makes sense to me. The user is visualizing the data in an index. 
Also, IMO a new user is more likely going understand what an "Index data visualizer" is going to do vs a "Data view data visualizer".

Reply:
Changed in https://github.com/elastic/kibana/pull/124125/commits/8ac0b4e349c7f17e079d9add1100fd26f2e33670